src/test/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientTest.java

```diff
@@ -329,4 +355,44 @@ public void testGetHealthStatus() {
     assertEquals(healthStatus, client.getHealthStatus());
 
   }
+
+  @Test
+  public void testGetSupportedVersions() {
+    stubActiveVersionsResponse();
+
+    BasicChatbotClient client = ChatbotClients.basic()
+        .basePath(basePath)
+        .authMechanism(mockAuthMechanism)
+        .build();
+
+    ((BasicChatbotClientImpl) client).setVersionsApi(mockVersionsApi);
+
+    assertEquals(1, client.getSupportedVersions().getVersions().size());
+  }
+
+  private void stubActiveVersionsResponse() {
+    List<SupportedVersionsVersions> versions = new ArrayList<>();
+    SupportedVersionsVersions version = new SupportedVersionsVersions();
+    version.setVersionNumber("5.0.0");
+    version.setStatus(StatusEnum.ACTIVE);
+    versions.add(version);
+    SupportedVersions mockVersions = new SupportedVersions();
+    mockVersions.setVersions(versions);
+    Mono<SupportedVersions> monoResponse = Mono.fromCallable(() -> mockVersions);
+    when(mockVersionsApi.versionsGet()).thenReturn(monoResponse);
+    ((BasicChatbotClientImpl) client).setVersionsApi(mockVersionsApi);
+  }
+
+  private void stubInactiveVersionsResponse() {
+    List<SupportedVersionsVersions> versions = new ArrayList<>();
+    SupportedVersionsVersions version = new SupportedVersionsVersions();
+    version.setVersionNumber("5.1.0");
+    version.setStatus(StatusEnum.ACTIVE);
+    versions.add(version);
+    SupportedVersions mockVersions = new SupportedVersions();
```

Comment:
Can you refactor to avoid duplicate code in test methods?

Reply:
Yes